ddtrace/settings/_otel_remapper.py

```diff
@@ -0,0 +1,184 @@
+import os
+
+from ..internal.logger import get_logger
+
+
+log = get_logger(__name__)
+
+
+OTEL_UNIFIED_TAG_MAPPINGS = {
+    "deployment.environment": "DD_ENV",
+    "service.name": "DD_SERVICE",
+    "service.version": "DD_VERSION",
+}
+
+
+def _remap_otel_log_level(otel_value):
+    """Remaps the otel log level to ddtrace log level"""
+    if otel_value == "debug":
+        return "True"
+    elif otel_value == "info":
+        return "False"
+    else:
+        log.warning(
+            "ddtrace does not support otel log level '%s'. setting ddtrace to log level info.",
+            otel_value,
+        )
+        return "False"
+
+
+def _remap_otel_propagators(otel_value):
+    """Remaps the otel propagators to ddtrace propagators"""
+    accepted_styles = []
+    for style in otel_value.split(","):
+        style = style.strip().lower()
+        if style in ["b3", "b3multi", "b3single", "datadog", "tracecontext", "none"]:
+            if style == "b3single":
+                style = "b3"
+            if style not in accepted_styles:
+                accepted_styles.append(style)
+        else:
+            log.warning("Following style not supported by ddtrace: %s.", style)
+    return ",".join(accepted_styles)
+
+
+def _remap_traces_sampler(otel_value):
+    """Remaps the otel trace sampler to ddtrace trace sampler"""
+    if otel_value == "always_on" or otel_value == "parentbased_always_on":
+        return "1.0"
+    elif otel_value == "always_off" or otel_value == "parentbased_always_off":
+        return "0.0"
+    elif otel_value == "traceidratio" or otel_value == "parentbased_traceidratio":
+        return os.environ.get("OTEL_TRACES_SAMPLER_ARG", "1")
+    else:
+        return otel_value
+
+
+def _remap_traces_exporter(otel_value):
+    """Remaps the otel trace exporter to ddtrace trace exporter"""
+    if otel_value != "none":
+        log.warning("An unrecognized trace exporter '%s' is being used; setting DD_TRACE_ENABLED to False.", otel_value)
+        return "False"
+    else:
+        return "True"
```

Comment:
This appears to be a bug, if the value is `none` we should still be returning `"False"` since the user configured that they wanted to turn off traces

Reply:
Good catch. We only support the `none` configuration. All other configurations (ex: otlp,zipkin,console) should not influence ddtrace. Since `DD_TRACE_ENABLED` is `True` by default returning `True` here actually doesn't do anything. 

```suggestion
    return "False"
```